lua/nvim-lsp-installer/server.lua

```diff
@@ -66,6 +67,28 @@ function M.Server:get_default_options()
     return vim.deepcopy(self._default_options)
 end
 
+function M.Server:get_supported_filetypes()
```

Comment:
At first glance it looks like this would not capture any filetypes that is overridden by users when calling `.setup {}`.

Reply:
I'm not sure where you store that info. My idea was to just avoid doing the lookup more than once.

Reply:
This plugin doesn't store any information regarding how the servers are set up. The `Server:setup(opts)` function is just a proxy for lspconfig's `setup` functions. Not sure how they store it internally, but I'd imagine there's no public APIs for it